js/src/credentialResponse/types/ldpVc/LdpVcCredentialResponse.js

```diff
@@ -0,0 +1,10 @@
+const CredentialResponse = require("../../CredentialResponse");
+
+class LdpVcCredentialResponse extends CredentialResponse {
+
+    toJsonString(response) {
+        return JSON.parse(response);
```

Comment:
The return type of the method is object but the name looks like we are returning a stringified object, can we please look into the names?

`
> typeof JSON.parse("{}")
'object'
> typeof JSON.stringify({})
'string'
> typeof JSON.stringify("{}")
'string'`